src/L1GovernorMetadataBridge.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {IGovernor} from "openzeppelin/governance/Governor.sol";
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+contract L1GovernorMetadataBridge {
+  /// @notice The governor where proposals are fetched and bridged.
+  IGovernor public governor;
```

Comment:
nit: can be immutable, right?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/8/commits/5211bb94a9b648a59dbedd1b6cb22f116421c456